module1.js

```diff
@@ -96,6 +115,9 @@ function question6(array) {
 
 function question7(array1, array2, extraElement) {
   // TODO:
+  var twoArrays = [...array1, ...array2];
```

Comment:
Good! 
Also, what function on an array object do you know that would do the same thing? :)

Reply:
Concat